src/app/pages/listing/listing-edit-upload.jsx

```diff
@@ -457,24 +320,150 @@ function ChplListingEditUploadPage({ id }) {
                 </CardContent>
               </Card>
             </ListingContext.Provider>
-          </div>
-          <div>
-            { newListing
-              && (
-                <ListingContext.Provider value={newListingState}>
-                  <Card>
-                    <Box className={classes.sectionHeader}>
-                      <Typography className={classes.sectionHeaderText} variant="h2">Additional Information</Typography>
-                    </Box>
-                    <CardContent>
-                      <ChplAdditionalInformation
-                        listing={newListing}
-                      />
-                    </CardContent>
-                  </Card>
-                </ListingContext.Provider>
+          </Box>
+          </Box>
+          <Box>{/* Upload Container */}
+            <Typography gutterBottom component={"h3"} style={{ fontWeight: '600' }} variant='h4'>Updated Listing</Typography>
+            <div>
+              {!newListing ? (
+                <Box className={classes.placeholderContainer}>
+                  <HelpOutlineIcon fontSize="large" color="primary" />
+                  <Typography>Upload a file above to display your new listing.</Typography>
+                </Box>
+              ) : (
+                <>
+                  {newListing && (
+                    <Fade style={{ transitionDelay: newListing ? '1s' : '0ms' }} in={!!newListing}>
```

Comment:
This is a similar problem. To even get to this part, the `newListing` _must_ exist, so doing logic based on the existence of the `newListing` is n/a